tools/scripts/ffi.toml

```diff
@@ -312,4 +313,137 @@ else
     echo "Installing Diplomat rev ${rev}"
     exec cargo install --git https://github.com/rust-diplomat/diplomat.git --rev ${rev} diplomat-tool -f
 end
-'''
\ No newline at end of file
+'''
+
+[tasks.gn-install]
```

Comment:
@Manishearth Please pay special attention to the changes in this file to make sure this all looks sane